src/interceptors/WebSocket/index.ts

```diff
@@ -136,7 +140,7 @@ export class WebSocketInterceptor extends Interceptor<WebSocketEventMap> {
                 socket.readyState !== WebSocket.CLOSING &&
                 socket.readyState !== WebSocket.CLOSED
               ) {
-                socket.close(1000, error.message)
+                socket[kClose](1000, error.message, false)
```

Comment:
This is the only use case when the interceptor will produce a non-clean connection closure.